easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4_configure.patch

```diff
@@ -0,0 +1,29 @@
+--- ./configure.ac	2017-04-04 11:47:18.067209892 +0930
```

Comment:
@nathanhaigh please add a short description at the top of the patch file, and mention who created it (or include a reference to upstream fix)